Tests/api.test.js

```diff
@@ -0,0 +1,120 @@
+    expect(response.statusCode).toBe(200);
+    expect(response.text.length).toBeGreaterThan(0);
+  });
+
+  it("should return updated todo, when updated todo exists", async () => {
+    const todo = await driver.when
+      .addTodo(randomStr())
+      .then((r) => JSON.parse(r.text));
+    todo.completed = true;
+    const request = await driver.when.updateTodo(todo);
+    expect(request.status).toBe(200);
+    expect(JSON.parse(request.text)).toEqual(todo);
+  });
+
+  it("should return error, when updated todo doesn't exist", async () => {
+    const todo = {
+      title: randomStr(),
+      id: randomStr(),
+      completed: false,
+    };
+    const request = await driver.when.updateTodo(todo);
+    expect(request.status).toBe(400);
+  });
+
+  it("should return error, when updated todo was not passed", async () => {
+    const request = await driver.when.updateTodo();
+    expect(request.status).toBe(400);
+  });
+
+  it("should return deleted todo, when todo with passed id exists", async () => {
+    const todo = await driver.when
+      .addTodo(randomStr())
+      .then((r) => JSON.parse(r.text));
+    const request = await driver.when.removeTodo(todo.id);
+    expect(JSON.parse(request.text)).toEqual(todo);
+    const deletedTodo = await driver.when.getTodo(todo.id);
+    expect(deletedTodo).toBeUndefined();
+  });
+
+  it("should return error, when deleted todo doesnt exist", async () => {
+    const todo = {
+      title: randomStr(),
+      id: randomStr(),
+      completed: false,
+    };
+    const request = await driver.when.removeTodo(todo.id);
+    expect(request.status).toBe(400);
+  });
+
+  it("should return array with todos", async () => {
+    const response = await driver.request().get("/todo");
+    expect(response.status).toBe(200);
+    expect(JSON.parse(response.text)).toEqual([]);
+  });
+
+  it("should return completed todos, when COMPLETED filter is passed", async () => {
+    const todo = await driver.when
+      .addTodo(randomStr())
+      .then((r) => JSON.parse(r.text));
+    todo.completed = true;
+    await driver.when.updateTodo(todo);
+    const request = await driver.request().get("/todo?filter=COMPLETED");
+    expect(JSON.parse(request.text)).toEqual([todo]);
+  });
+
+  it("should return uncompleted todos, when UNCOMPLETED filter is passed", async () => {
+    const todo = await driver.when
+      .addTodo(randomStr())
+      .then((r) => JSON.parse(r.text));
+    await driver.when.updateTodo(todo);
+    const request = await driver.request().get("/todo?filter=UNCOMPLETED");
+    expect(JSON.parse(request.text)).toEqual([todo]);
+  });
+
+  it("should return all todos, when ALL filter is passed", async () => {
+    const todo = await driver.when
+      .addTodo(randomStr())
+      .then((r) => JSON.parse(r.text));
+    const request = await driver.request().get("/todo?filter=ALL_TASKS");
+    expect(JSON.parse(request.text)).toEqual([todo]);
+  });
```

Comment:
Can you please update code, so it will have 2 todos. 
One completed and one not completed